code_challenges/hacker_rank/compress_the_string.py

```diff
@@ -0,0 +1,18 @@
+def compressing_strings(a_string):
+    count = 0
+    current_letter = a_string[0]
+    
+    for letter in a_string:
+        if letter == current_letter:
+            count += 1
+        else:
+            print(f'({count}, {current_letter}) ', end = '')
+            current_letter = letter
+            count = 1
+print(f'({count}, {current_letter})', end = '')
```

Comment:
This global print statement should either be in the if name is main block or removed.